gui-tk/minsky.tcl

```diff
@@ -707,6 +707,12 @@ proc logVarsOK {} {
 .menubar.edit add command -label "Group selection" -command "minsky.createGroup" -accelerator $meta_menu-G
 .menubar.edit add command -label "Dimensions" -command dimensionsDialog
 
+proc togglePaste {window} {
+	if {[catch {clipboard get -type UTF8_STRING}]} {
+	.menubar.edit entryconfigure "Paste" -state disabled
```

Comment:
Excellent - maybe we should use "Paste" instead of end for the toggleGodleyPaste proc as well...